dashboard/app/jobs/concerns/ai_rubric_metrics.rb

```diff
@@ -43,6 +43,29 @@ def self.log_metric(metric_name:, agent: nil, value: 1)
     )
   end
 
+  # Log thumbs up / down feedback for AI evaluation
+  # @param [LearningGoalAiEvaluationFeedback] feedback
+  def self.log_feedback(feedback)
+    lesson = feedback.learning_goal_ai_evaluation&.learning_goal&.rubric&.lesson
+
+    Cdo::Metrics.push(
+      AI_RUBRIC_METRICS_NAMESPACE,
+      [
+        {
+          metric_name: 'LearningGoalAiEvaluationFeedback',
+          value: 1,
+          dimensions: [
+            {name: 'Environment', value: CDO.rack_env},
+            {name: 'Unit', value: lesson&.script&.name || ''},
+            {name: 'Lesson', value: lesson&.relative_position&.to_s || ''},
+            {name: 'Approval', value: feedback.ai_feedback_approval ? 'thumbs_up' : 'thumbs_down'}
+          ],
+          unit: 'Count',
+        }
+      ]
```

Comment:
We should think about demensions carefully here. Excess or poorly designed dimensions can create noise and increase cost. I initially wrote a novel here, but [this section in the docs](https://docs.aws.amazon.com/AmazonCloudWatch/latest/monitoring/cloudwatch_concepts.html#Dimension) explains it clearly, so make sure you're happy with these dimensions. I don't have a perfect mental model, so I think my only concern is whether changes to Units could throw off the lesson numbers and impact historic metrics - like if a lesson gets added or removed from the middle of a Unit.

For other reader reference, here's how this looks as data starts to get populated:
![image](https://github.com/user-attachments/assets/f204a985-4991-47ed-b6d3-6ad1661b0b23)

Reply:
per offline discussion, we do not want to log unit and lesson because we just plan to alert on the (already low) total volume of thumbs up and thumbs down events. if we want to do deeper analysis, we can look at the data in redshift and even set up a tableau dashboard if we want to. additionally, including these dimensions would come 
with some cost, because aggregating across all possible values for a dimension in cloudwatch is difficult.